pandas/core/generic.py

```diff
@@ -7791,9 +7791,9 @@ def between_time(
         self: NDFrameT,
         start_time,
         end_time,
-        include_start: bool_t | lib.NoDefault = lib.no_default,
-        include_end: bool_t | lib.NoDefault = lib.no_default,
-        inclusive: IntervalClosedType | None = None,
```

Comment:
This is slightly different, None is not a valid value, so we should not even accept now it otherwise when we remove the deprecated keyword argument, we may end up with users that have explicitly passed None.